src/test/java/com/googleinterns/zoomtube/utils/CommentUtilTest.java

```diff
@@ -46,67 +46,98 @@ public void tearDown() {
   }
 
   @Test
-  public void createComment_shouldReturnCommentFromEntity() throws IOException {
+  public void createComment_reply_shouldReturnCommentFromEntity() throws IOException {
     Key lecture = KeyFactory.createKey(LectureUtil.KIND, 12345);
     Key parent = KeyFactory.createKey(CommentUtil.KIND, 67890);
-    long timestampMs = 123;
     User author = new User("test@example.com", "example.com");
     String content = "Test content";
     Date dateNow = new Date();
     Entity entity = new Entity(CommentUtil.KIND);
+    Comment.Type type = Comment.Type.REPLY;
     entity.setProperty(CommentUtil.LECTURE, lecture);
     entity.setProperty(CommentUtil.PARENT, parent);
-    entity.setProperty(CommentUtil.TIMESTAMP_MS, timestampMs);
     entity.setProperty(CommentUtil.AUTHOR, author);
     entity.setProperty(CommentUtil.CONTENT, content);
     entity.setProperty(CommentUtil.CREATED, dateNow);
+    entity.setProperty(CommentUtil.TYPE, type.toString());
 
     Comment comment = CommentUtil.createComment(entity);
 
     assertThat(comment.commentKey()).isEqualTo(entity.getKey());
     assertThat(comment.lectureKey()).isEqualTo(lecture);
     assertThat(comment.parentKey().isPresent()).isTrue();
     assertThat(comment.parentKey().get()).isEqualTo(parent);
-    assertThat(comment.timestampMs()).isEqualTo(timestampMs);
     assertThat(comment.author()).isEqualTo(author);
     assertThat(comment.content()).isEqualTo(content);
     assertThat(comment.created()).isEqualTo(dateNow);
+    assertThat(comment.type()).isEqualTo(type);
   }
 
   @Test
-  public void createEntity_noParent_shouldReturnEntityWithProperties_noParent() throws IOException {
+  public void createComment_root_shouldReturnCommentFromEntity() throws IOException {
+    Key lecture = KeyFactory.createKey(LectureUtil.KIND, 12345);
+    long timestamp = 123;
```

Comment:
please make this a global constant as well

Reply:
Done

Reply:
This doesn't need to be a global const. You can even use it raw which is was is preferred. Please look at (Local Variables and Class-Wide Constants): https://docs.google.com/document/d/18T9ef-DRqTncVD377GoIQmOGI84cNKkZ7Nn07YURFTw/edit#